BlazorApp1/Pages/Index.razor

```diff
@@ -1,726 +1,102 @@
 ﻿@page "/"
 @using Microsoft.JSInterop;
-
+@inject IJSRuntime JSRuntime
 @inject FusionChartsService fusionChartsService;
-
 <PageTitle>Index</PageTitle>
```

Comment:
meaningful title

Reply:
I've changed the title to Column2D Chart